graphrag/index/workflows/v1/create_final_relationships.py

```diff
@@ -18,19 +22,11 @@ def build_steps(
     * `workflow:create_base_entity_graph`
     * `workflow:create_final_nodes`
     """
-    base_text_embed = config.get("text_embed", {})
-    relationship_description_embed_config = config.get(
-        "relationship_description_embed", base_text_embed
-    )
-    skip_description_embedding = config.get("skip_description_embedding", False)
+    log.info(config)
```

Comment:
Same. I would remove any logging that is put in for dev/debug and not useful for regular runs of the system